athena-dynamodb/pom.xml

```diff
@@ -33,6 +33,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-sts</artifactId>
+            <version>1.12.440</version>
```

Comment:
For the version, use this instead:
`<version>${aws-sdk.version}</version>`

Reply:
ack